packages/@angular/cli/blueprints/ng/index.ts

```diff
@@ -78,6 +91,18 @@ export default Blueprint.extend({
       fileList = fileList.filter(p => p.indexOf('app.component.spec.ts') < 0);
     }
 
+    if (this.options && this.options.barebones) {
+      fileList = fileList.filter(p => p.indexOf('e2e/') < 0);
```

Comment:
Can you chain those and use Regexes? I think it'd be simpler to read.

```typescript
fileList = fileList.filter(p => {
  return !(   p.match(/e2e\//)
           || p.match(/editorconfig/)
           || p.match(/README\.md/)
           || /* ... */)
```

Opened to suggestions, I just don't think `indexOf` here is not that legible.